asset-transfer-basic/rest-api-typescript/src/server.ts

```diff
@@ -0,0 +1,67 @@
+import helmet from 'helmet';
+import { StatusCodes, getReasonPhrase } from 'http-status-codes';
+import express, { NextFunction, Request, Response } from 'express';
+import pino from 'pino';
+import pinoMiddleware from 'pino-http';
+
+const logger = pino({
+  level: process.env.LOG_LEVEL || 'info',
+});
+
+const app = express();
+const { BAD_REQUEST, INTERNAL_SERVER_ERROR, OK } = StatusCodes;
+
+app.use(pinoMiddleware({
+  logger,
+  customLogLevel: function customLogLevel(res, err) {
+    if (res.statusCode >= BAD_REQUEST && res.statusCode < INTERNAL_SERVER_ERROR) {
+      return 'warn';
+    }
+
+    if (res.statusCode >= INTERNAL_SERVER_ERROR || err) {
+      return 'error';
+    }
+
+    return 'debug';
+  },
+}));
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+if (process.env.NODE_ENV === 'development') {
+  // TBC
+}
+
+if (process.env.NODE_ENV === 'production') {
+  app.use(helmet());
+}
+
+app.get('/ready', (_req, res) => res.status(OK).json({
```

Comment:
Nice to have these to be able to route K8S probes here.

Reply:
That was one of the recommendations from https://github.com/nodeshift/nodejs-reference-architecture which seemed like a good plan